backend/app/api/v1/commons/utils.py

```diff
@@ -133,3 +134,25 @@ def buildReleaseStreamFilter(input_array):
         )
         mapped_array.append(match)
     return list(set(mapped_array))
+
+
+def get_dict_from_qs(query_string):
+    query_dict = parse_qs(query_string)
+    cleaned_dict = {
+        key: [v.strip("'") for v in values] for key, values in query_dict.items()
+    }
+
+    return cleaned_dict
+
+
+def construct_query(filter_dict):
+    query_parts = []
+    if isinstance(filter_dict, dict):
+        for key, values in filter_dict.items():
+            k = constants.FIELDS_FILTER_DICT[key]
+            if len(values) > 1:
+                or_clause = " or ".join([f'{k}="{value}"' for value in values])
+                query_parts.append(f"{or_clause}")
```

Comment:
This is really just `query_parts.append(or_clause)` 😆

Reply:
You didn't change this? On the one hand, it won't break anything, but it's an awkward usage that'll unnecessarily draw the attention of a reader away from the code flow. 😆

Reply:
It got missed. Now, updated it :grin: